### PR TITLE
add recipe for georaster

### PR DESCRIPTION
**Pull request by atedstone**

### CODE REVIEW

File: recipes/georaster/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+test:
+  imports:
+    - georaster
+
+about:
+  home: https://github.com/atedstone/georaster
+  license: GNU GPL v.3
+  summary: 'easy processing and analysis of geographic and projected rasters'
+  license_family: GPL3
+  license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/georaster/meta.yaml]: Change this to: `license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'`. It's currently looking in the source directory from the download.

File: recipes/georaster/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  md5: 72f2e8c57aee8cb01b689025f75a0d90
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
```
Comment (synapticarbors) [inline recipes/georaster/meta.yaml]: It does not look like `numpy` ,`scipy`, `gdal` and `pyproj` are required to build the recipe.

File: recipes/georaster/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+{% set version = "1.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 72f2e8c57aee8cb01b689025f75a0d90
```
Comment (synapticarbors) [inline recipes/georaster/meta.yaml]: Please switch to the `sha256` hash

**Comment by atedstone** [comment]
Thanks for review @synapticarbors . Now having unforeseen problems with __init__.py. Will fix after holidays.

**Comment by scopatz** [comment]
Please enable CircleCI on your fork of staged recipes: http://conda-forge.github.io/docs/ciservices.html#circleci-linux

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**